src/AggregateFunctions/AggregateFunctionTimeSeriesGroupSum.h

```diff
@@ -189,15 +189,21 @@ struct AggregateFunctionTimeSeriesGroupSumData
     {
         size_t size = result.size();
         writeVarUInt(size, buf);
-        buf.write(reinterpret_cast<const char *>(result.data()), sizeof(result[0]));
+        if (size > 0)
+        {
+            buf.write(reinterpret_cast<const char *>(result.data()), sizeof(result[0]));
```

Comment:
It does not look consistent with `deserialize`.

Reply:
indeed, thanks...